config/files/usr/etc/security/authramp.conf

```diff
@@ -0,0 +1,25 @@
+# AuthRamp Configuration File
+# This file configures the behavior of the AuthRamp PAM module.
+#
+[Configuration]
+# Directory where tally information is stored.
+# Each user has a separate file in this directory to track authentication failures.
+# tally_dir = /var/run/authramp
+#
+# Number of allowed free authentication attempts before applying delays.
+# During these free tries, the module allows authentication without introducing delays.
+# free_tries = 6
+#
+# Base delay applied to each authentication failure.
+# This is the initial delay applied after the free tries are exhausted.
+# base_delay_seconds = 30
```

Comment:
Are these included as a comment just to document the defaults?

Reply:
Yes this is how the configuration file is installed. I quite like the usability when packages do this since it documents the config and shows the default values.

Reply:
The only change from default is the even_deny_root flag to also block brute-force attempts on the root account